packages/lesswrong/lib/collections/comments/views.js

```diff
@@ -222,6 +222,17 @@ ensureIndex(Comments, {legacyId: "hashed"});
 // Used in scoring cron job
 ensureIndex(Comments, {inactive:1,postedAt:1});
 
+Comments.addView("sunshineNewUsersComments", function (terms) {
+  return {
+    selector: {
+      userId: terms.userId,
+      $or: null,
+    },
+    options: {sort: {postedAt: -1}, limit: terms.limit || 5},
+  };
+});
+// TODO – ask Jim about indices
```

Comment:
Yep, this needs its own index.

Reply:
I didn't know how to account for the $or being blank. Does that mean the index just needs "userID: 1"?